build/configs/artik053/st_things/defconfig

```diff
@@ -811,6 +830,7 @@ CONFIG_ELOOP_POLL=y
 # CONFIG_WPA_SUPPLICANT_CMD is not set
 CONFIG_DRIVER_T20=y
 # CONFIG_ENABLE_EAP_FOR_SUPPLICANT is not set
+# CONFIG_WIFI_PROFILE_SECURESTORAGE is not set
```

Comment:
Why did you turn off this flag?
Don't you need to encrypt WiFi information?

Reply:
Yes, this config needs to be enabled.